bitpack/numbers.go

```diff
@@ -0,0 +1,92 @@
+package bitpack
+
+import (
+	"unsafe"
+)
+
+const (
+	stringEncUin64DictLen   = 62
+	stringEncUint64MaxBytes = 11
+)
+
+var (
+	encodeLookup = [62]byte{
```

Comment:
Correct me if I'm wrong:
```suggestion
	encodeLookup = [stringEncUin64DictLen]byte{
```

Reply:
Fair point, forgot that when I recently shortened the array - good catch!